community/docker/APKBUILD

```diff
@@ -127,12 +136,13 @@ package() {
 	install -Dm755 "$_containerd_builddir"/bin/ctr \
 		"$pkgdir"/usr/bin/docker-containerd-ctr || return 1
 
-	install -Dm755 bundles/$ver/dynbinary/dockerinit-$ver \
-		"$pkgdir"/usr/lib/docker/dockerinit
+	install -Dm755 "$_tini_builddir"/tini-static \
+		"$pkgdir"/usr/bin/docker-init || return 1
+
 	install -Dm755 contrib/init/openrc/docker.initd \
-		"$pkgdir"/etc/init.d/docker
+		"$pkgdir"/etc/init.d/docker || return 1
```

Comment:
Leonardo wrote:
> +build() {
> +     cd "$builddir"
> +     make || return 1
> +}

We don't need any more to use "|| return 1". Please see
http://lists.alpinelinux.org/alpine-devel/5608.html

> +
> +package() {
> +     cd "$builddir"
> +     make DESTDIR="$pkgdir" install || return 1
> +}
> +

http://lists.alpinelinux.org/alpine-aports/4391.html